### ISSUE TITLE
🐛 BUG: Very inconsistent formatting in default example projects

### ISSUE THREAD

**Issue by MarcusOtter** [withastro/astro#3855]
### What version of `astro` are you using?

v1.0.0-beta.64

### Are you using an SSR adapter? If so, which one?

No

### What package manager are you using?

npm

### What operating system are you using?

Windows

### Describe the Bug

Not sure if this is the right place for this, but it felt better than making an RFC (happy to move it somewhere else though)

All of the example projects' formatting are very inconsistent, which is not a great user experience. The first file I opened look like this, just to give an example from the `Just the basics` template (linked repro).
|  ![image](https://user-images.githubusercontent.com/35617441/177894473-881a492c-13ac-4b4a-b83b-edee976d2e09.png) |  ![image](https://user-images.githubusercontent.com/35617441/177894532-56431da1-2360-41d6-b420-6786b564628e.png) |  ![image](https://user-images.githubusercontent.com/35617441/177894721-e4f923b3-cf42-44b1-9e59-4157e3e2f6e1.png) |
| --- | --- | --- |
| Inconsistent indentation (tabs vs spaces) | Mixed between ending self-closing tags with `/` or not | Mix between semicolons and commas, and inconsistent spacing & typing (two different components) |

I got formatting working using the wonderful [prettier-plugin-astro](https://github.com/withastro/prettier-plugin-astro), but users shouldn't have to deal with setting this up, the defaults should be sane even if they don't want prettier. I know the prettier plugin is in the early stages (and that's probably why `.astro` files are exlcuded in the root `.prettierignore`), but I have two questions:

# TLDR
Formatting non-existent in examples.

1. When can we expect to automatically format the entire monorepo (with all `.astro` files) with prettier?
2. Is there any interest to get just the examples decently formatted before that happens?

I have gotten the formatter running in the astro repo locally so I could submit a PR with only the format changes (without hooking up the plugin) if there's any interest. Well, I could PR hooking up the plugin too 😄 

I'll CC @Princesseuh on this one!

### Link to Minimal Reproducible Example

https://stackblitz.com/github/withastro/astro/tree/latest/examples/basics?file=src/layouts/Layout.astro

### Participation

- [X] I am willing to submit a pull request for this issue.

**Comment by Princesseuh** [comment]
Ha! I have an issue on my board this week exactly about this

My plan was to fix this issue: https://github.com/withastro/prettier-plugin-astro/issues/193 and then run Prettier with the plugins on all the examples. 

The reason I wanted to wait for that issue first was because whenever that error happen, it's considered a fail in CI which wouldn't be too convenient to have in the monorepo (though I'm not sure if any of the examples trigger that issue at the moment or not? It has a few ways to be triggered)

**Comment by MarcusOtter** [comment]
Created a PR here #3862 @Princesseuh
